content/en/observability_pipelines/installation.md

```diff
@@ -110,6 +108,85 @@ $ DD_API_KEY=<DD_API_KEY> DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.
 [7]: /observability_pipelines/configurations/
 [8]: /observability_pipelines/working_with_data/
 {{% /tab %}}
+{{% tab "RPM-based Linux" %}}
+
+## Overview
+
+Install the Observability Pipelines Worker with the [RPM Package Manager][1] (RPM), a free package manager that handles the installation and removal of software on [RedHat Enterprise Linux][2], [CentOS][3], and other [Linux][4] distributions.
+
+## Prerequisites
+
+Before installing, make sure you:
+
+1. Are using one of the supported Linux architectures: x86_64 or AMD64
+2. Have a valid [Datadog API key][5].
+3. Have an [Observability Pipelines Configuration][6].
+
+## Installation
+
+### Manual
+
+1. Run the following commands to set up the Datadog `rpm` repo on your system:
+
+    ```
+    cat <<EOF > /etc/yum.repos.d/datadog-observability-pipelines-worker.repo
+    [observability-pipelines-worker]
+    name = Observability Pipelines Worker
+    baseurl = https://yum.datadoghq.com/stable/observability-pipelines-worker-1/x86_64/
+    enabled=1
+    gpgcheck=1
+    repo_gpgcheck=1
+    gpgkey=https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public
+           https://keys.datadoghq.com/DATADOG_RPM_KEY_FD4BF915.public
+    EOF
+    ```
+
+**Note:** If yo uare running RHEL/CentOS 8.1, use `repo_gpgcheck=0` instead of `repo_gpgcheck=1` in the configuration above.
```

Comment:
```suggestion
   **Note:** If you are running RHEL 8.1 or CentOS 8.1, use `repo_gpgcheck=0` instead of `repo_gpgcheck=1` in the configuration above.
```